docs/sdk/auth/fragments/android/working-with-api.md

```diff
@@ -118,7 +118,27 @@ AWSMobileClient.getInstance().signIn(username, password, null, new Callback<Sign
 });
 ```
 
-## Confirm SignIn
+### Confirm SignIn (MFA)
+
+In order to setup multi-factor authentication, choose `Manual configuration` while setting up auth in the CLI and when you get to the `Multifactor authentication` step, choose these values:
+
+```bash
+ Multifactor authentication (MFA) user login options: ON (Required for all logins, can not be enabled later)
+ For user login, select the MFA types: SMS Text Message
+ Please specify an SMS authentication message: Your authentication code is {####}
+ Email based user registration/forgot password: Enabled (Requires per-user email entry at registration)
+ Please specify an email verification subject: Your verification code
+ Please specify an email verification message: Your verification code is {####}
+ Do you want to override the default password policy for this User Pool? No
+ Warning: you will not be able to edit these selections. 
+ What attributes are required for signing up? Email, Phone Number (This attribute is not supported by Facebook, Login With Amazon.)
+```
+
+Note that for the `What attributes are required for signing up?` prompt you currently need to use the arrow keys to scroll down in the list and select `Phone Number`. Otherwise you will not be able to add a phone number to the user and thus will not be able to sign in since SMS MFA is required.
```

Comment:
Nit: Hmm, you noted this above in the example output. Is this an added caveat that is the user neglects to follow the directions, auth will not work? Perhaps we can be terser in the language here?